### PR TITLE
hw-mgmt: infra: Add initial support NVIDIA HGX Vulcan Baseboard

### PR DESCRIPTION
**Pull request by vadimp-nvidia**
Add basic infrastructure for new VMOD0019.
Add skeleton for global write protect.

Signed-off-by: Vadim Pasternak <vadimp@nvidia.com>

### CODE REVIEW

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

File: usr/usr/bin/hw-management-global-wp.sh [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+##################################################################################
+# Copyright (c) 2020 - 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Comment (felixradensky) [inline usr/usr/bin/hw-management-global-wp.sh:3]: The copyright should be 2020-2022

File: usr/usr/bin/hw-management-global-wp.sh [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		case $1 in
+		release)
+			while [ "$global_wp_timeout" -gt 0 ]
+			do
+				# Request to disable Global Write Protection.
+				# Write Global Write Protection request.
+				echo 1 > "$system_path"/global_wp_request
+
+				# Validate if request to disable Global Write Protection was accepted.
+       	 			sleep 0.1 &
```
Comment (felixradensky) [inline usr/usr/bin/hw-management-global-wp.sh:53]: The indentation should be fixed

**Pull request closed without merging**